### PR TITLE
Fix an Insight complaint about .travis.yml

### PR DESCRIPTION
**Pull request by dunglas**
| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | n/a
| License       | MIT
| Doc PR        | n/a

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -28,9 +28,7 @@ matrix:
 
 before_install:
     - composer self-update
-    - if [ "$SYMFONY_VERSION" != "" ]; then
```
Comment (theofidry) [inline .travis.yml]: What's the issue? Looks far more readable to have in on multiple lines IMO. And Insight YAML checker doesn't apply to the `.travis.yml` as Travis is parsing the YAML a different way.

Comment (dunglas) [inline .travis.yml]: https://insight.sensiolabs.com/projects/a93f5a40-483f-4c46-ba09-3e1033b62552/analyses/564?status=violations#364261812

Comment (theofidry) [inline .travis.yml]: Travis removes the line return character so it is parsed as `if [ "$SYMFONY_VERSION" != "" ]; then composer require "symfony/symfony:${SYMFONY_VERSION}" --no-update; fi;`.

I would rather remove `.travis.yml` from Insight check.

Comment (sroze) [inline .travis.yml]: Crappy Insight :stuck_out_tongue: Happy to merge as I don't care, that's readable even on 1 line.

**Comment by sroze** [comment]
:+1:

**Pull request merged**